CONTRIBUTING.md

```diff
@@ -86,11 +86,11 @@ IGNORE IMAGE ANALYSIS
 ```
 
 | Rule | Description | Notes / Examples |
-|-|-|-|
-| `INVERT` | Inverts specified elements. | **Dynamic Mode**: INVERT only for dark images that are invisible on dark backgrounds. |
-| `CSS` | Adds custom CSS to a web page. | `!important` keyword should be specified for each CSS property to prevent overrides by other stylesheets.<br>**Dynamic mode** supports `${COLOR}` template, where `COLOR` is a color value before the inversion. <br>*Example*: `${white}` will become `${black}` in dark mode. |
-| `IGNORE INLINE STYLE` | Prevents inline style analysis of matched elements. | *Example*: `<p style="color: red">` element's style attribute will not be changed. |
-| `IGNORE IMAGE ANALYSIS` | Prevents background images from being analyzed for matched selectors. |  |
+|---|---|---|
+| **INVERT** | Inverts specified elements. | **Dynamic Mode**: INVERT only for dark images that are invisible on dark backgrounds. |
+| **CSS** | Adds custom CSS to a web page. | `!important` keyword should be specified for each CSS property to prevent overrides by other stylesheets.<br>**Dynamic mode** supports `${COLOR}` template, where `COLOR` is a color value before the inversion. <br>*Example*: `${white}` will become `${black}` in dark mode. |
+| **IGNORE&nbsp;INLINE&nbsp;STYLE** | Prevents inline style analysis of matched elements. | *Example*: `<p style="color: red">` element's style attribute will not be changed. |
+| **IGNORE&nbsp;IMAGE&nbsp;ANALYSIS** | Prevents background images from being analyzed for matched selectors. |  |
```

Comment:
`&nbsp;` I'd find this a very very hacky way.

Reply:
Me too.  Do you know a better technique?  I tried about 10 different techniques and spent over 30 minutes reading documentation and other resources before I settled on `&nbsp;`.  Ideally, one could just add a little CSS, but every resource I could find says that GitHub prohibits it.

Reply:
BTW, I copied the result into a text editor and was pleased that the resulting spaces are true ASCII 0x20 spaces.

I couldn't find an equivalent non-breaking character for dashes, as everything that looks like a dash (but isn't really one) will not be ASCII  0x2D.

I wanted copy-paste to work for people and not mess up the destination file with characters that looked correct visually but weren't actually the correct ASCII value.

Reply:
Well, let it be then, it indeed seems to be the only way.